cmd/buildctl/build.go

```diff
@@ -46,7 +46,7 @@ var buildCommand = cli.Command{
 		},
 		cli.StringFlag{
 			Name:  "progress",
-			Usage: "Set type of progress (auto, plain, tty). Use plain to show container output",
+			Usage: "Set type of progress (auto, plain, tty, raw:json). Use plain to show container output",
```

Comment:
My intention is that `raw` would be the format and `json` is the microformat for raw. Maybe it would be better to just label it `raw` and then add `raw:json` in the future if we ever decided to do other formats for the raw type.

I feel like `events` probably won't be descriptive enough.